paddle/fluid/pybind/pybind.cc

```diff
@@ -1007,6 +1000,10 @@ All parameter, weight, gradient are variables in Paddle.
           "memory_optimize",
           [](const BuildStrategy &self) { return self.memory_optimize_; },
           [](BuildStrategy &self, bool b) { self.memory_optimize_ = b; })
+      .def_property(
+          "is_distribution",
```

Comment:
Maybe `is_distribution` is not a suitable field, how about `update_method` which type is enum including `pserver, nccl2` ...

Reply:
If the program is pserver mode or num_trainers is greater than 1, the `is_distribution` should be true, is it ok?